### DISCUSSION TITLE
Who is super?

### DISCUSSION THREAD

**Discussion by Superblue045** [WebXDAO/start-here#504] [General]
Hi everyone. I am Hiroshi. Nice to meet u. I'm new here as well.

**Comment by shuklaritvik06** [comment]
Hello,

Thank you for reaching out! We are an open-source community dedicated to fostering growth and collaboration. If you want to contribute to our community, we encourage you to join and you can also start your own project in our banner. We can provide our banner and give you a shout-out on our socials to help you gain more employment opportunities.

To join us and get started, please join our Discord community here: https://discord.webxdao.xyz/.

Looking forward to your contributions!

Best,
Ritvik

**Comment by Superblue045** [comment]
Thanks
